src/components/Login/LoginForm.tsx

```diff
@@ -0,0 +1,151 @@
+import { NUMBER_REGEX } from '@/constants/regex'
+import theme from '@/styles/theme'
+import DESIGN_TOKEN from '@/styles/token'
+import { Button, TextField, Typography } from '@mui/material'
+import React from 'react'
+
+interface Props {}
+
+const LoginForm: React.FC<Props> = () => {
+  const [inputValues, setInputValues] = React.useState({
+    id: '',
+    password: '',
+  })
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
+    const { name, value } = event.target
+    setInputValues((prevValues) => ({
+      ...prevValues,
+      [name]: value,
+    }))
+  }
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>): void => {
+    event.preventDefault()
+    console.log(inputValues)
+  }
+
+  return (
+    <form onSubmit={handleSubmit} style={{ flex: 1, width: '520px', fontFamily: 'Noto Sans Thai' }}>
+      <TextField
+        id="id"
+        name="id"
+        label="รหัสนักศึกษา"
+        variant="outlined"
+        inputProps={{
+          maxLength: 11,
+          pattern: '[0-9]*',
+          title: 'รหัสนักศึกษาต้องเป็นตัวเลขเท่านั้น',
+        }}
+        sx={{
+          '& .MuiOutlinedInput-root': {
+            color: theme.palette.primary.main,
+          },
+          '& .MuiInputLabel-root': {
+            color: inputValues.id ? theme.palette.primary.main : DESIGN_TOKEN.color.primary[6],
+            fontSize: '18px',
+          },
+          '& .MuiInputLabel-root.Mui-focused': {
+            color: theme.palette.primary.main,
+            fontSize: '18px',
+          },
+          '& .MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-notchedOutline': {
+            borderColor: theme.palette.primary.main,
+            fontSize: '18px',
+          },
+          '& .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline': {
+            borderColor: inputValues.id
+              ? theme.palette.primary.main
+              : DESIGN_TOKEN.color.primary[6],
+            fontSize: '18px',
+          },
+          width: '100%',
+          fontSize: '24px',
+          height: '95px',
+        }}
```

Comment:
Create custom component for TextField in theme config or create new component. dont hardcode inline style. Use theme variable for fontsize